homeassistant/components/buienradar/sensor.py

```diff
@@ -60,132 +61,272 @@
 # Supported sensor types:
 # Key: ['label', unit, icon]
 SENSOR_TYPES = {
-    "stationname": ["Stationname", None, None],
+    "stationname": ["Stationname", None, None, None],
     # new in json api (>1.0.0):
-    "barometerfc": ["Barometer value", None, "mdi:gauge"],
+    "barometerfc": ["Barometer value", None, "mdi:gauge", None],
     # new in json api (>1.0.0):
-    "barometerfcname": ["Barometer", None, "mdi:gauge"],
+    "barometerfcname": ["Barometer", None, "mdi:gauge", None],
     # new in json api (>1.0.0):
-    "barometerfcnamenl": ["Barometer", None, "mdi:gauge"],
-    "condition": ["Condition", None, None],
-    "conditioncode": ["Condition code", None, None],
-    "conditiondetailed": ["Detailed condition", None, None],
-    "conditionexact": ["Full condition", None, None],
-    "symbol": ["Symbol", None, None],
+    "barometerfcnamenl": ["Barometer", None, "mdi:gauge", None],
+    "condition": ["Condition", None, None, None],
+    "conditioncode": ["Condition code", None, None, None],
+    "conditiondetailed": ["Detailed condition", None, None, None],
+    "conditionexact": ["Full condition", None, None, None],
+    "symbol": ["Symbol", None, None, None],
     # new in json api (>1.0.0):
-    "feeltemperature": ["Feel temperature", TEMP_CELSIUS, "mdi:thermometer"],
-    "humidity": ["Humidity", PERCENTAGE, "mdi:water-percent"],
-    "temperature": ["Temperature", TEMP_CELSIUS, "mdi:thermometer"],
-    "groundtemperature": ["Ground temperature", TEMP_CELSIUS, "mdi:thermometer"],
-    "windspeed": ["Wind speed", SPEED_KILOMETERS_PER_HOUR, "mdi:weather-windy"],
-    "windforce": ["Wind force", "Bft", "mdi:weather-windy"],
-    "winddirection": ["Wind direction", None, "mdi:compass-outline"],
-    "windazimuth": ["Wind direction azimuth", DEGREE, "mdi:compass-outline"],
-    "pressure": ["Pressure", PRESSURE_HPA, "mdi:gauge"],
-    "visibility": ["Visibility", LENGTH_KILOMETERS, None],
-    "windgust": ["Wind gust", SPEED_KILOMETERS_PER_HOUR, "mdi:weather-windy"],
+    "feeltemperature": [
+        "Feel temperature",
+        TEMP_CELSIUS,
+        "mdi:thermometer",
+        DEVICE_CLASS_TEMPERATURE,
+    ],
+    "humidity": ["Humidity", PERCENTAGE, "mdi:water-percent", None],
+    "temperature": [
+        "Temperature",
+        TEMP_CELSIUS,
+        "mdi:thermometer",
+        None,
+        DEVICE_CLASS_TEMPERATURE,
```

Comment:
This item number 5. Is that correct? Looks like we access index 3, ie item number 4 in the property.

Reply:
Fixed in 8e16641